packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -537,14 +577,143 @@ class _ListWheelScrollViewState extends State<ListWheelScrollView> {
             clipToSize: widget.clipToSize,
             renderChildrenOutsideViewport: widget.renderChildrenOutsideViewport,
             offset: offset,
-            children: widget.children,
+            childDelegate: widget.childDelegate,
           );
         },
       ),
     );
   }
 }
 
+/// Element that supports building children lazily for [ListWheelViewport].
+class ListWheelElement extends RenderObjectElement {
+  /// Creates an element that lazily builds children for the given widget.
+  ListWheelElement(ListWheelViewport widget) : super(widget);
+
+  @override
+  ListWheelViewport get widget => super.widget;
+
+  @override
+  RenderListWheelViewport get renderObject => super.renderObject;
+
+  /// A cache of widgets so that we don't have to rebuild every time.
+  final Map<int, Widget> _childWidgets = new HashMap<int, Widget>();
+
+  /// The map containing all active child elements.
```

Comment:
To be honest I use SplayTreeMap because the original sliver code uses it too : )) To look again I think that SplayTreeMap is appropriate because we need an ordered map.

Reply:
I can't figure out why the original implementation uses SplayTreeMap by reading the doc vs the default LinkedHashMap. 

https://stackoverflow.com/questions/14930950/what-are-the-differences-between-the-different-map-implementations-in-dart is sensible (because of the self balancing root move optimization). If it's applicable here (frequent access), it's worth documenting.

Reply:
Agree, I will add more explanation.

Reply:
Sounds reasonable